pkgs/development/tools/language-servers/fennel-ls/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, stdenv, fetchgit, lua, luaPackages }:
```

Not necessary, but it would make the package a lot more diff friendly.

```suggestion
{ lib
, stdenv
, fetchgit
, lua
, luaPackages
}:
```